Absorb ASVS as draft software security standards

Celtikill opened the pull request:
initial edits, to format ASVS to easy reading and use.

jkirbysecurity reviewed (approved):
Awesome job, thanks for adding this to our program!

Going forward, I'd like to get in the habit of adding a PR Description in the top set of comments.  PR Descriptions should include:

- The purpose for the pull request 
- A summary of what is being added, removed, or changed 
- Links to associated tracking issues and/or previous conversations 
- Steps for testing/validating changes/fixes 

## Approved!

Celtikill commented:
> Awesome job, thanks for adding this to our program!
> 
> Going forward, I'd like to get in the habit of adding a PR Description in the top set of comments. PR Descriptions should include:
> 
>     * The purpose for the pull request
> 
>     * A summary of what is being added, removed, or changed
> 
>     * Links to associated tracking issues and/or previous conversations
> 
>     * Steps for testing/validating changes/fixes
> 
> 
> ## Approved!

great idea, I'll add this prior to PR merge

jkirbysecurity commented on system_requirements/INDEX.md:
What's the difference between **System Requirements** and **Validation Requirements**?

Celtikill replied:
I made the decision to remove all the extra, so the question is now moot :)

#ReduceReuseRecycle

Celtikill commented:
> Awesome job, thanks for adding this to our program!
> 
> Going forward, I'd like to get in the habit of adding a PR Description in the top set of comments. PR Descriptions should include:
> 
>     * The purpose for the pull request
> 
>     * A summary of what is being added, removed, or changed
> 
>     * Links to associated tracking issues and/or previous conversations
> 
>     * Steps for testing/validating changes/fixes
> 
> 
> Couple other suggestions...
> 
>     * add a `CODEOWNERS` file
> 
>     * One of my favorite resources for devs.  Could we somehow reference/integrate the OWASP Top 10 Proactive Controls -- https://top10proactive.owasp.org/
> 
>     * One of my other favorite resources for devs.  Could we somehow reference/integrate the OWASP Cheat Sheet Series -- https://cheatsheetseries.owasp.org/
> 
> 
> ### Approved!

I've added CODEOWNERS file (@jkirbysecurity and myself as co-owners of all in repo).  I think we should merge this PR asap, then iterate on the (excellent) changes suggested in proactive control and cheatsheet mapping.  #Iterate
